documentation/api/io.strimzi.api.kafka.model.KafkaMirrorMakerSpec.adoc

```diff
@@ -45,9 +47,13 @@ spec:
     type: inline
     loggers:
       mirrormaker.root.logger: "INFO"
```

Comment:
This is for MM1, which is deprecated. This is a minor change so it's fine, but I think we should avoid doing non-security changes.